stylus/components/nav.styl

```
@@ -62,14 +62,12 @@ $nav-item-icon
 
     :hover > &
         color var(--charcoalGrey)
-
     +medium-screen()
         display block
-        font-size rem(24)
         margin-right 0
 
         svg
-            margin 0 auto
+            margin 0 auto 3px auto
```

```suggestion
            margin 0 auto 3px
```

will fix the red travis